CLAUDE.md

```
@@ -0,0 +1,83 @@
+
+# Build container image
+make docker-build IMG=<registry>/<image>:<tag>
+```
+
+> Tests use `controller-runtime`'s envtest harness. `testbin/` is populated automatically on first `make test`.
+
+---
+
+## Repo Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)